ext/CreateParametersExt.jl

```diff
@@ -166,7 +166,9 @@ function FarquharParameters(
     parameters = CP.get_parameter_values(toml_dict, name_map, "Land")
     FT = CP.float_type(toml_dict)
     MECH = typeof(mechanism)
-    return FarquharParameters{FT, MECH}(;
+    Vcmax25 = FT.(Vcmax25)
```

Comment:
do we need this FT? If Vcmax25 is a field, it should already be made on the space of the simulation and be the right type. If it is a scalar, this definition in the FarquharParameter struct `VC <: Union{FT, ClimaCore.Fields.Field}` should ensure it is the same type as the other scalars.

Reply:
I think Julia does not perform conversion from scalars to unions. If I remove line 169 the tests fail with 
` TypeError: in FarquharParameters, in VC, expected VC<:Union{Float32, ClimaCore.Fields.Field}, got Type{Float64}`. Additionally, if I try converting from a scalar to `Union{FT, ClimaCore.Fields.Field}` I get 
`MethodError: Cannot convert an object of type 
  Float64 to an object of type 
  Union{Float32, ClimaCore.Fields.Field}
`